core/src/cubos/core/data/file.cpp

```diff
@@ -68,19 +69,19 @@ void File::mount(std::string_view path, std::shared_ptr<Archive> archive)
     {
         if (dir->parent != nullptr)
         {
-            logError("Could not mount archive at path '{}', a file already exists at that path", dir->path);
+            CUBOS_CRITICAL("Could not mount archive at path '{}', a file already exists at that path", dir->path);
             abort();
         }
 
         if (dir->archive != nullptr)
         {
-            logError("Could not mount archive at root, an archive has already been mounted at root");
+            CUBOS_CRITICAL("Could not mount archive at root, an archive has already been mounted at root");
```

Comment:
I'd remove the ending `at root` to avoid repetition:
```suggestion
            CUBOS_CRITICAL("Could not mount archive at root, an archive has already been mounted");
```